pkgs/development/python-modules/aiopvapi/default.nix

```diff
@@ -18,8 +18,8 @@ buildPythonPackage rec {
     owner = "sander76";
     repo = "aio-powerview-api";
     # no tags on git, no sdist on pypi: https://github.com/sander76/aio-powerview-api/issues/12
```

Comment:
The comment should have been removed and this is a home-assistant leaf dependency the complete change was already in #190221